app/views/static/payment.html.erb

```diff
@@ -0,0 +1,92 @@
+<html>
+<meta name="viewport" content="width=device-width, initial-scale=1" />
+<head>
+<%= csrf_meta_tags %>
+<script src="https://js.stripe.com/v3/"></script>
+<script>
+  function stripeTokenHandler(token) {
```

Comment:
right now none of this javascript is going through webpack. Here is what I normally do:

1. Create a `/components` folder in `app/javascripts`
1. Include the file in `application.js` (just like `channels`
1. Create a file called `stripe-form.js` in `app/javascripts/components`
1. Paste this javascript into that component file.

Note this javascript will run on every page so you will need to make sure it works without the elements on the page.